parakeet_rocm/cli.py

```
@@ -0,0 +1,504 @@
+
+    """
+    # Delegation to heavy implementation (lazy import)
+    from importlib import import_module  # pylint: disable=import-outside-toplevel
+
+    # Normalise default
+    if audio_files is None:
+        audio_files = []
+
+    # Validate input combinations
+    if not audio_files and not watch:
+        raise typer.BadParameter("Provide AUDIO_FILES or --watch pattern(s).")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)